spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -63,6 +71,14 @@ def get_value(self, name):
             raise
         except CommError:
             raise ValueError(msg % reason_comm)
+        except ModuleNotFoundError as e:
+
+            if is_conda_based_app():
+                raise ValueError(msg % reason_missing_package
+                                 .format(module = e.name))
+            else:
+                raise ValueError(msg % reason_missing_package_conda
+                                 .format(module = e.name))
```

Comment:
```suggestion
            if is_conda_based_app():
                raise ValueError(
                    msg % reason_missing_package_installer.format(e.name)
                )
            else:
                raise ValueError(
                    msg % reason_missing_package.format(e.name)
                )
```

- To go with my suggestions above
- Improve code style.